flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -88,8 +88,11 @@ run_test "Resuming Externalized Checkpoint after terminal failure (rocks, increm
 # Docker
 ################################################################################
 
-run_test "Running Kerberized YARN on Docker test (default input)" "$END_TO_END_DIR/test-scripts/test_yarn_kerberos_docker.sh"
-run_test "Running Kerberized YARN on Docker test (custom fs plugin)" "$END_TO_END_DIR/test-scripts/test_yarn_kerberos_docker.sh dummy-fs"
+# Ignore these tests on Azure
```

Comment:
why?

Reply:
In these tests, the TaskManagers are not starting on YARN, probably due to memory constraints.
Do you agree to file a JIRA ticket to fix this?

Reply:
of course, but also document in the file why they are excluded

Reply:
Ok --> https://issues.apache.org/jira/browse/FLINK-15833